ksqldb-common/src/main/java/io/confluent/ksql/serde/WindowInfo.java

```diff
@@ -73,12 +85,14 @@ public boolean equals(final Object o) {
       return false;
     }
     final WindowInfo that = (WindowInfo) o;
+    // we omit `emitStrategy`
```

Comment:
Why do we omit this from the comparator?

Reply:
`WindowInfo` should actually only describe the window itself and some code relies on it -- emit-strategy is not really a window property and it's just bolted on her as it's hard to get a hold on it otherwise.

Because `emit-strategy` could be set on one instance but not on another, while both instances describe the same window, we omit it here to not break the comparison.

(IIRC)